### ISSUE TITLE
ci: master container image doesn't point an images which is built from the master branch

### ISSUE THREAD

**Issue by satoru-takeuchi** [rook/rook#6181]
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**

rook/`<provider>`:master container image in Dockerhub should point the one which is built from the master branch. However, in Fri Aug 28 01:20:11 UTC 2020,  this tag points release-1.3 branch's image can be rook/`<provider>`:master.

- "master" tag
  - digest 6b60c80f14dd (linux/amd64)
- "v1.3.0.950.gbebaed0" tag
  - digest 6b60c80f14dd (linux/amd64)

In addition, the example manifests in the master branch uses "rook/`<provided>`:master" image.

https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/operator.yaml#L275

So if use it, a v1.3 operator is used by accident.

**Expected behavior:**

rook/`<provider>`:master image point the one which is built from the master branch. Or create other appropriate tags.

**How to reproduce it (minimal and precise):**

Please see the dockerhub.
https://hub.docker.com/r/rook/ceph/tags

**File(s) to submit**:

* Cluster CR (custom resource), typically called `cluster.yaml`, if necessary
* Operator's logs, if necessary
* Crashing pod(s) logs, if necessary

 To get logs, use `kubectl -n <namespace> logs <pod name>`
When pasting logs, always surround them with backticks or use the `insert code` button from the Github UI.
Read [Github documentation if you need help](https://help.github.com/en/articles/creating-and-highlighting-code-blocks).

**Environment**:
* OS (e.g. from /etc/os-release):
* Kernel (e.g. `uname -a`):
* Cloud provider or hardware configuration:
* Rook version (use `rook version` inside of a Rook Pod):
* Storage backend version (e.g. for ceph do `ceph -v`):
* Kubernetes version (use `kubectl version`):
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift):
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/rook/master/ceph-toolbox.html)):

**Comment by travisn** [comment]
@satoru-takeuchi The `master` tag is actually correct and the tags you referred to are the latest master branch, but there is a known issue with the tags... When we push the `master` tag, we also push a tag based off master that is generated by a git command that says how many commits since the last label in master. Usually the last label off master would be the last release. Now, we would expect that the last label was 1.4.0-beta.0 when we created the 1.4 branch. However, the 1.4.0-beta.0 label was created on a later commit in the release-1.4 branch after we had committed the manifest version changes. So now the master tag goes all the way back to the 1.3 tag, which is confusing. I'm going to look at adding a tag that will fix this issue...
